### PR TITLE
extension: add more matched 4399 sites for content scripting

### PR DESCRIPTION
**Pull request by riku-42**
This PR is a part of work for #17386

### CODE REVIEW

**Comment by Dinnerbone** [comment]
I'm a *little* hesitant here, but it's not your fault - this is good, it's just that we've discovered that browsers now show ruffle as "needing permissions on <all sites> and also <each domain listed here>". It's a little ridiculous, but it's already caused some people to make assumptions and go "what's 4399? Why is ruffle talking to some chinese site?"

Adding more will make that list even scarier, maybe? Can we get away with something like `*.4399.com/*`?

**Review by Dinnerbone — APPROVED** [review]

File: web/packages/extension/src/background.ts [hunk 1]
```diff
@@ -193,7 +193,12 @@ async function enable() {
             },
             {
                 id: "4399",
-                matches: ["https://www.4399.com/flash/*"],
+                matches: [
+                    "*://www.4399.com/flash/*",
+                    "https://news.4399.com/qiu/",
+                    "https://my.4399.com/*",
+                    "http://sjsj.4399.com/",
```
Comment (torokati44) [inline web/packages/extension/src/background.ts:200]: Is the presence/absence of a `*` at the end meaningful and intentional, or accidental? :thinking:

Comment (riku-42) [reply web/packages/extension/src/background.ts:200]: The absence of a * is intentional because that is the game URL!😀

Comment (Korne127) [reply web/packages/extension/src/background.ts:200]: Maybe group the general URLs with the wildcards and the game URLs together each? (basically switching lines 198 and 199)? But that's a minor nitpick; this generally looks good :)

Comment (riku-42) [reply web/packages/extension/src/background.ts]: that's a nice nitpick!

**Review by Korne127 — APPROVED** [review]

**Review by torokati44 — COMMENTED** [review]
I was only asking for clarification, but I agree that it's indeed nicer grouped this way, thank you!
I'm late with the "please squash on merge" note, but oh well. 🙃

**Pull request merged**